local-interchain/go.mod

```diff
@@ -1,50 +1,49 @@
 module github.com/strangelove-ventures/localinterchain
 
-go 1.19
+go 1.21
 
 replace (
-	github.com/ChainSafe/go-schnorrkel => github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d
-	github.com/ChainSafe/go-schnorrkel/1 => github.com/ChainSafe/go-schnorrkel v1.0.0
-	github.com/btcsuite/btcd => github.com/btcsuite/btcd v0.22.2 //indirect
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+	github.com/misko9/go-substrate-rpc-client/v4 => github.com/faddat/go-substrate-rpc-client/v4 v4.0.1-0.20240402155230-48db8c110afe
```

Comment:
cc @misko9 will defer to you here with these changes

https://github.com/ComposableFi/go-substrate-rpc-client/compare/master...faddat:go-substrate-rpc-client:faddat/fix-go-substrate-forinterchaintest

Reply:
Works for me. We just need a place with all the required changes for Composable and then, the latest dependencies which @faddat's branch now has.